src/services/videomania/index.jsx

```
@@ -248,3 +255,205 @@ export const AddVideoMania = ({
+              onClick={handleLike}
+              className="action-button"
+              disabled={isLoading}
+            >
+              <IoHeart className={`w-6 h-6 ${isLiked ? "text-red-500" : ""}`} />{" "}
+              {likes}
+            </button>
+
+            <button
+              className="action-button"
+              onClick={() => setIsModalOpen(true)}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)